### PR TITLE
Added IGMPv3 Setters and Supporting Code

### PR DESCRIPTION
**Pull request by jlshuman1961**
Added Setters to MaxResponseTime and QueriersQueryInterval, and changed their types to ushort to handle converted values appropriately.
Added supporting methods:
    getHighestOneBit()
    ConvertFloatingPointToCode()

### CODE REVIEW

**Comment by jlshuman1961** [comment]
Chris Morgan sent me this in an email (but I didn't know how to do it):

"It should be pretty easy to up rev the library version, CI will push to nuget automatically.

I’d make that request once you’ve got the other things resolved. You could even open a pr with the changes to bump the rev to speed things along.

Chris"

File: PacketDotNet/IgmpV3MembershipQueryPacket.cs [hunk 1]
```diff
@@ -218,6 +232,45 @@ private byte CodeOrFloatingPointValue(byte code)
             return (byte) ((mant | 0x10) << (exp + 3));
         }
 
+        private int getHighestOneBit(int n)
```
Comment (PhyxionNL) [inline PacketDotNet/IgmpV3MembershipQueryPacket.cs:235]: Capitalize.

File: PacketDotNet/IgmpV3MembershipQueryPacket.cs [hunk 2]
```diff
@@ -67,7 +67,14 @@ public short Checksum
                                                                Header.Bytes);
 
         /// <summary>Gets or sets the IGMPv3 membership query max response time, in tenths of seconds.</summary>
-        public byte MaxResponseTime => CodeOrFloatingPointValue(MaxResponseCode);
+        public ushort MaxResponseTime
+        {
+            get => CodeOrFloatingPointValue(MaxResponseCode);
```
Comment (PhyxionNL) [inline PacketDotNet/IgmpV3MembershipQueryPacket.cs:72]: Something I missed the previous one, can we rename this to GetCodeOrFloatingPointValue.
Also change to expression body style (similar to the get here). Also for the property below.

**Review by PhyxionNL — CHANGES_REQUESTED** [review]

File: PacketDotNet/IgmpV3MembershipQueryPacket.cs (hunk 1 above)
Comment (jlshuman1961) [reply PacketDotNet/IgmpV3MembershipQueryPacket.cs]: Done!

File: PacketDotNet/IgmpV3MembershipQueryPacket.cs (hunk 2 above)
Comment (jlshuman1961) [reply PacketDotNet/IgmpV3MembershipQueryPacket.cs]: Done!
And added missing return for floatValue < 128 in ConvertFloatingPointToCode().

Comment (PhyxionNL) [reply PacketDotNet/IgmpV3MembershipQueryPacket.cs]: Can you also rename CodeOrFloatingPointValue (see above comment)?

Comment (jlshuman1961) [reply PacketDotNet/IgmpV3MembershipQueryPacket.cs]: Done! 
And, also, corrected the return type on one of those methods.

**Comment by jlshuman1961** [comment]
Does this mean that you can now bump the rev so that a new version of PacketDotNet can be released with the IGMPv3 support?

**Comment by PhyxionNL** [comment]
Yes, we can bump after needed changes in #160.

**Pull request merged**